pkg/util/progressdots/progressdots.go

```diff
@@ -45,15 +54,20 @@ func (s *ProgressDots) Start() {
 				return
 			default:
 				fmt.Print(".")
+				s.dotCounter++
 				time.Sleep(s.interval)
+				if s.dotCounter%10 == 0 {
```

Comment:
Not sure yet how this works. In my opinion the terms are not easy to understand, so the code does not describe itself well. In electronics and motion we have something called 'easing' which can slow down a motion or value according to a curve. Although we are not describing not a real curve here, the effect is similar.

To me, `dotCounter` only has an internal respresentation to keep a record of the dots I have written since the last easing. 

```go
  	return &ProgressDots{
 		interval:   DefaultInterval,
  		easing:     DefaultEasing,
 		handler:    make(chan struct{}),
  	}

...

s.dotCounter++
if easing != 0 && s.dotCounter % easing == 0 {
   s.dotCounter = 0
   s.interval = s.interval + 1 * time.Second
}
```

Reply:
@gbraad Somehow I completely missed to need a different variable to store the variable :(